strr-examiner-web/app/pages/examine/[applicationId].vue

```diff
@@ -33,19 +41,32 @@ const { data: application, status, error, refresh } = await useLazyAsyncData<
 
 const handleApplicationAction = (
   id: string,
-  action: 'APPROVE' | 'REJECT',
+  action: 'APPROVE' | 'REJECT' | 'SEND_NOC',
   buttonPosition: 'left' | 'right',
   buttonIndex: number
 ) => {
-  const actionFn = action === 'APPROVE' ? approveApplication : rejectApplication
+  let actionFn
+  if (action === 'SEND_NOC') {
+    actionFn = sendNoticeOfConsideration
+  } else if (action === 'APPROVE') {
+    actionFn = approveApplication
+  } else if (action === 'REJECT') {
+    actionFn = rejectApplication
+  }
+
+  const additionalArgs = action === 'SEND_NOC' ? [nocContent.value.content] : []
```

Comment:
This line 57 & 58 could be moved in the condition where the action is checked, below line 50.

Reply:
Cleaned up and it now more readable.